internal/plugins/ansible/v1/scaffolds/internal/templates/makefile.go

```diff
@@ -70,33 +70,43 @@ IMG ?= {{ .Image }}
 
 all: docker-build
 
-# Run against the configured Kubernetes cluster in ~/.kube/config
-run: ansible-operator
+##@ Targets
```

Comment:
This Makefile update is unrelated to bumping kb, can you move it to a separate PR

Reply:
sure we can do that in a follow-up.

Reply:
It is not. We need to change the manifests scaffolds for that and then, it is used for all types which mean that this PR requires to ship the help feature for all.

Reply:
This change should be aligned with https://github.com/kubernetes-sigs/kubebuilder/pull/1981